awx/main/models/projects.py

```diff
@@ -28,6 +28,7 @@
     UnifiedJob,
     UnifiedJobTemplate,
 )
+from awx.main.models.jobs import Job
```

Comment:
this might have actually been a bug

https://github.com/AlanCoding/awx/blob/fbf6315a8cf62a45149c7ac36a019088acbdd1e3/awx/main/models/projects.py#L505

I don't think this could have come from models.base

Reply:
@chrismeyersfsu I searched and the method `is_blocked_by` is never used, so that's why the missing import never caused a problem. Do you have any insight about how it was called by non-trivial means? Or maybe it would be better to delete the method.